homeassistant/components/qnap_qsw/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "invalid_id": "Device returned an invalid unique ID"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "url": "[%key:common::config_flow::data::url%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "description": "Set up QNAP QSW integration."
```

Comment:
I think we can drop this since it doesn't add much value and puts more work on translators

```suggestion
```

Reply:
Done in e2d35bf30a4c7067dd36f9520f903b326de660b5